### PR TITLE
[traefik] added ssl.cipherSuites, imageTag and chart version bumped

### PR DESCRIPTION
**Pull request by dennybaa**
**What this PR does / why we need it**:

Adds an ability to specify custom **cipherSuites**.

### CODE REVIEW

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -52,6 +52,7 @@ ssl:
   enforced: false
   upstream: false
   insecureSkipVerify: false
+  cipherSuites: []
```
Comment (dtomcej) [inline stable/traefik/values.yaml]: Can you comment this out?

File: stable/traefik/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.41.0
-appVersion: 1.6.5
+version: 1.41.1
```
Comment (dtomcej) [inline stable/traefik/Chart.yaml]: Can you rebase from master?

**Review by dtomcej — CHANGES_REQUESTED** [review]

File: stable/traefik/values.yaml (hunk 1 above)
Comment (dennybaa) [reply stable/traefik/values.yaml]: yup, sure I can. But does this change anything? Basically the explanation in the README is that I've managed to come up with. Fell free to change it. Imo it should be clear enough.
PS. I've made the new commit commenting the above out.

File: stable/traefik/Chart.yaml (hunk 2 above)
Comment (dennybaa) [reply stable/traefik/Chart.yaml]: yup sorry missed it

**Comment by dtomcej** [comment]
@dennybaa  It seems you also need to sign off on the commits to pass the DCO check.

**Comment by dennybaa** [comment]
@dtomcej what is dco and how to sign up?

**Comment by dtomcej** [comment]
`git commit --amend --no-edit -s` to sign off on the commit. Then `git push -f` to repush it (I think)

**Comment by dennybaa** [comment]
yup done thanks

**Comment by dennybaa** [comment]
@dtomcej @emilevauge just a friendly reminder, dco is signed, so can be merged. Just needs your approval. Thank you!

**Comment by dtomcej** [comment]
@mattfarina Can you give this PR a love bump? Thanks!

**Review by dtomcej — APPROVED** [review]

**Comment by dennybaa** [comment]
again please /lgtm

**Pull request merged**